### DISCUSSION TITLE
Supported Audio Formats and Pre-processing Checks in whisper

### DISCUSSION THREAD

**Discussion by ghost** [openai/whisper#1700] [Q&A]
Hello pyannote maintainers and community,

I'm currently integrating whisper into a project and was wondering about its audio compatibility. Specifically:

1. **Supported Formats:** Which audio formats (e.g., .wav, .mp3, .flac, etc.) are supported by whisper out of the box? I'd like to know so that I can ensure the audio files I use are compatible.

2. **Pre-processing Check:** Does whisper perform any pre-checks on the audio files before processing them? For instance, if an unsupported or corrupted audio file is passed, does whisper identify this and raise an error or warning?

Having clarity on these aspects will be very helpful as I look to integrate whisper more seamlessly into my workflows.

Thank you for your assistance and for the great work on this tool!

Best regards,

**Comment by phineas-pta** [comment]
it should work with widely used format supported by ffmpeg like wav mp3

if corrupted file it simply crash u need proper error handling
